data.sql

```diff
@@ -0,0 +1,10 @@
+
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Agumon', '2020-02-03', 0, 'true', 10.23);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', '2018-11-15', 2, 'true', 8.00);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', '2021-01-07', 1, 'false', 15.04);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '2017-05-12', 5, 'false', 11);
```

Comment:
- [ ] For `Devimon` the value for `neutered` should be set to `true` as per requirement.
![image](https://github.com/ragangithub/vet-clinic-db/assets/81932478/48b7393a-fd5a-4f82-bdc6-60e1447110aa)
check requirements [here](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md)

## Important 🔴 
### Note that the screenshots have to be udated for all queries where changes are required